apps/src/dropletUtils.js

```diff
@@ -131,11 +131,14 @@ standardConfig.categories = {
  * @param {DropletConfig} dropletConfig
  * @returns {Array}
```

Comment:
comment added. I was calling it standardConfig, but realized that it is a global in the same file and I didn't like passing a parameter where the same variable name was in scope locally and globally.